### DISCUSSION TITLE
Error first time running build_runner

### DISCUSSION THREAD

**Discussion by DanBaDo** [simolus3/drift#2716] [Q&A] [answered]
Hi!
I'm trying to use Drift for first time.
I'm following Drift web setup documentation.
When I try to run build_runner, I got a warning about three errors and `database.g.dart` is not created:
![image](https://github.com/simolus3/drift/assets/85959254/15822327-3e8d-4cb1-82b2-89328706a6b3)

- database.dart:22:24: A class declaration must have a body, even if it is empty.
- database.dart:22:25: Expected a method, getter, setter or operator declaration.
- database.dart:22:26: Functions must have an explicit list of parameters.

This is my code: 
```dart
import 'dart:io';

import 'package:drift/drift.dart';
import 'package:drift/native.dart';

import 'package:path/path.dart' as path;

part 'database.g.dart';

@DataClassName('TodoCategory')
@TableIndex(name: 'user_email', columns: {#email})
@TableIndex(name: 'user_uuid', columns: {#uuid})
class UserTable extends Table {
  TextColumn get uuid => text().withLength(min: 37, max: 37).unique()();
  TextColumn get email => text().unique()();
  BoolColumn get enabled => boolean().withDefault( const Constant(true))();
  @override
  Set<Column> get primaryKey => {uuid};
}

@DriftDatabase(tables: [UserTable])
class Database extends _&Database {}
```

I can't see what I'm doing wrong.

**Comment by kuhnroyal** [comment, marked as answer]
`_&Database` looks wrong. Probably should be `_$Database`. Also better give your `Database` class a prefix so it doesn't clash with other classes called `Database`.  ` class FooDatabase extends _$FooDatabase {}`

**Reply by DanBaDo** [reply]
Oh, my!
Thank you, very, very much, and sorry about.
I've lost a few hours because of the `&`.
Thank you.
